data import/Data import.ipynb

```diff
@@ -0,0 +1,1333 @@
+       "    'promotion_price': '1849',\n",
+       "    'promotion_start': '20171023000000',\n",
+       "    'rrp': '2199',\n",
+       "    'vat_rate': '21'}},\n",
+       "  'title': {'5': 'Reax 8 TR 41',\n",
+       "   '6': 'Reax 8 TR 41',\n",
+       "   'C': 'Reax 8 TR 41',\n",
+       "   'H': 'Reax 8 TR 41',\n",
+       "   'L': 'Reax 8 TR 41',\n",
+       "   'Q': 'Reax 8 TR 41'}},\n",
+       " 'variant_id': '1070393001'}"
+      ]
+     },
+     "execution_count": 220,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Co to je promotion amount nevim je to neco jako sleva, je to u kazdyho, takze to muzeme brat jako indikator, ze je neco zlevneny\n",
+    "\n",
+    "jsou nektery params pro kazdou variantu stejny (treba type of shoes?) budeme muset prozkoumat, ktery veci jsou stejny napric vsema variantama.\n",
+    "\n",
+    "Nevim zatim jak presne funguje all year/season field, takze to tam zatim neni.. asi to bude kombinace toho jestli jsou summer nebo ne a toho co je tam o tom season type..\n",
+    "\n",
+    "Podobne nevim jak poznat pro koho jsou ty boty obecne, key determination, kde se prave pise mens nebo kids nekde chybi, takze to bude taky nejaky slozitejsi.. muzu to vykoukavat z dat, ale pokud nam to reknou bude to samozrejme rychlejsi.."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pak se koukam na to ktery keys v elems se vyskytujou vsude. Jsou to jen tri:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 206,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'COLOR', 'SIZE_EUR', 'TYPE_OF_SHOES'}"
+      ]
+     },
+     "execution_count": 206,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "l=[]\n",
+    "for line in data:\n",
+    "    a=set(line['variant_data']['params'].keys())\n",
+    "    l.append(a)\n",
+    "set.intersection(*l)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 235,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "numeric_vector = []\n",
+    "for line in data:\n",
+    "    numeric_vector.append({\n",
+    "        'user_id': line['user_id'],\n",
+    "        'product_id': line['product_id'],\n",
+    "        'variant_id': line['variant_id'],\n",
+    "        'date': line['date'],\n",
+    "        'brand': line['brand']['brand_id'],\n",
+    "        'price': int(line['variant_data']['price']['CZ1000']['price']),\n",
+    "        'promotion_amount': int(line['variant_data']['price']['CZ1000']['promotion_amount']),\n",
+    "        'Shoe_size': line['variant_data']['params']['SIZE_EUR'],\n",
+    "        'Type_of_shoes': line['variant_data']['params']['TYPE_OF_SHOES'],\n",
+    "        'Color': line['variant_data']['params']['COLOR']\n",
```

Comment:
Tady je asi nejvetsi neznama, a to ktera ta pole vybereme.

Reply:
presne tak, proto jsem tady vybiral ty co jsou jasny a smysluplny, k ostatnim bude fajn mit dokumentaci nebo jak jsem rikal je zavalit dotazama. Pripadne se to da samozrejme casto nejak vykoukat..